packages/@tinacms/react-core/CHANGELOG.md

```diff
@@ -7,165 +7,85 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ## [0.2.5-alpha.0](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.4...@tinacms/react-core@0.2.5-alpha.0) (2020-01-24)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.4](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.4-alpha.1...@tinacms/react-core@0.2.4) (2020-01-22)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.4-alpha.1](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.4-alpha.0...@tinacms/react-core@0.2.4-alpha.1) (2020-01-22)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.4-alpha.0](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.3...@tinacms/react-core@0.2.4-alpha.0) (2020-01-16)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.3](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.2...@tinacms/react-core@0.2.3) (2020-01-14)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.2](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.2-alpha.1...@tinacms/react-core@0.2.2) (2020-01-13)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.2-alpha.1](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.2-alpha.0...@tinacms/react-core@0.2.2-alpha.1) (2020-01-10)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.1](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.1-alpha.1...@tinacms/react-core@0.2.1) (2019-12-17)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 ## [0.2.1-alpha.1](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.1-alpha.0...@tinacms/react-core@0.2.1-alpha.1) (2019-12-17)
 
-
 ### Bug Fixes
 
-* **useForm:** allow forms in production ([c4bfebd](https://github.com/tinacms/tinacms/commit/c4bfebd))
-
-
-
-
+- **useForm:** allow forms in production ([c4bfebd](https://github.com/tinacms/tinacms/commit/c4bfebd))
 
 ## [0.2.1-alpha.0](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.0...@tinacms/react-core@0.2.1-alpha.0) (2019-12-06)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 # [0.2.0](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.2.0-alpha.0...@tinacms/react-core@0.2.0) (2019-12-02)
 
 **Note:** Version bump only for package @tinacms/react-core
 
-
-
-
-
 # [0.2.0-alpha.0](https://github.com/tinacms/tinacms/compare/@tinacms/react-core@0.1.1-alpha.0...@tinacms/react-core@0.2.0-alpha.0) (2019-11-28)
 
-
 ### Features
 
-* **useGlobalForm:** add hook for registering global forms ([d450cae](https://github.com/tinacms/tinacms/commit/d450cae))
-
-
-
-
+- **useGlobalForm:** add hook for registering global forms ([d450cae](https://github.com/tinacms/tinacms/commit/d450cae))
 
```

Comment:
Not sure if this file should be there ?

Reply:
This came from an odd merge but I've had a couple sets of eyes on it and we think everything looks okay.